test/EnsemblTranscriptDatasourceTest.py

```diff
@@ -218,6 +218,35 @@ def test_overlapping_gene_5flank(self):
         txs = ds.get_overlapping_transcripts("22", 22224920, 22224920)
         self.assertTrue(len(txs) == 0)
 
+    @TestUtils.requiresDefaultDB()
+    def test_5_prime_flank_annotation_positive_strand(self):
```

Comment:
How about a test that looks at 5' (both + and - transcripts) but at a distance just outside of the flank.  To make sure IGR is caught on an edge case.